istioctl/pkg/waypoint/waypoint.go

```diff
@@ -146,7 +153,38 @@ func Cmd(ctx cli.Context) *cobra.Command {
 				}
 				return err
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), "waypoint %v/%v applied\n", gw.Namespace, gw.Name)
+			progressLog := progress.NewLog()
+			manifestLog := progressLog.NewComponent(string(name.WaypointComponentName))
```

Comment:
I think the wait logic here is a little bit complex, and the `progressLog` is used for waiting on Istio components, which also changes the current display message. Maybe we could simplify this by just blocking and then outputting what we did previously when the gateway is ready, similar to `helm --wait`.

Reply:
I used the progressbar to be consistent with "istioctl install" behavior but if there is no preference for that, I am happy to switch to a simpler print.
Will change in the next commit.

Reply:
Done